evalml/automl/automl_search.py

```diff
@@ -836,34 +703,17 @@ def __init__(
         else:
             raise ValueError("Please specify a valid automl algorithm.")
 
+        self.allowed_model_families = [
```

Comment:
Added this logic for continuity but `self.allowed_model_families` is only used in search output and `self.allowed_pipelines` is used `__str__`.

Reply:
great call 😺